app/mailers/candidate_mailer.rb

```diff
@@ -283,69 +281,63 @@ def offer_accepted(application_choice)
   end
 
   def ucas_match_initial_email_duplicate_applications(application_choice)
-    @application_form = application_choice.application_form
     @course_name_and_code = application_choice.course_option.course.name_and_code
     @provider_name = application_choice.course_option.course.provider.name
     @date_to_withdraw_application_by = TimeLimitCalculator.new(rule: :ucas_match_candidate_withdrawal_request, effective_date: Time.zone.today).call.fetch(:time_in_future).to_s(:govuk_date)
 
     email_for_candidate(
-      @application_form,
+      application_choice.application_form,
       subject: I18n.t!('candidate_mailer.ucas_match_initial_email.duplicate_applications.subject'),
     )
   end
 
   def ucas_match_initial_email_multiple_acceptances(candidate)
-    @application_form = candidate.application_forms.first
     @date_to_withdraw_application_by = TimeLimitCalculator.new(rule: :ucas_match_candidate_withdrawal_request, effective_date: Time.zone.today).call.fetch(:time_in_future).to_s(:govuk_date)
 
     email_for_candidate(
-      @application_form,
+      candidate.application_forms.first,
```

Comment:
Should this be `candidate.current_application`?

Reply:
It shouldn't make any difference as the application_form is only used for retrieving the candidate name, but I have updated as it makes sense to always use the latest application data.